Problem1.java

```diff
@@ -0,0 +1,50 @@
+/**
+LeetCode Submitted : YES
+Space Complexity : O(target/Minimum of array)
+Time Complexity : O(2^length(of array))
+
```

Comment:
Please explain how you calculated the space complexity.

Reply:
This is incorrect; I was thinking in some other directions.
It should be number of digits only ?

Reply:
Yup will be O(n) space in worst case Prateek. Neha discussed this one with me